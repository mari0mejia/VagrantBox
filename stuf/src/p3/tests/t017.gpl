double a = 7 * 2;
double b = 7 * 4.2;
double d = 12.5 * 2;
double e = 12.5 * 4.2;
