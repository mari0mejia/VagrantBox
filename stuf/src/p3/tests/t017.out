gpl.cpp::main()
  input file(tests/t017.gpl)
  random seed(42)
  read_keypresses_from_standard_input(false)
  dump_pixels(false)
  symbol_table(true)
  graphics(false)

gpl.cpp::main() Calling yyparse()


gpl.cpp::main() after call to yyparse().

No errors found (parser probably worked correctly).


Printing the symbol table from main()
double a = 14
double b = 29.4
double d = 25
double e = 52.5
Graphics is turned off by the Makefile.  Program exiting.

gpl.cpp::main() done.
